gc_registry/certificate/routes.py

```diff
@@ -135,6 +136,12 @@ def certificate_bundle_cancellation(
     esdb_client: EventStoreDBClient = Depends(events.get_esdb_client),
 ):
     """Cancel a fixed number of certificates matched to the given filter parameters within the specified Account."""
+
+    # If no beneficiary is specified, default to the account holder
+    if certificate_bundle_action.beneficiary is None:
```

Comment:
Does a cancellation have to have a claimant? Not sure on the business logic here. How does this interface with the claim process?

Reply:
Claim definintely needs more fleshing out - I'm interpreting the standard as the gc bundle may be cancelled on behalf of the account or a third-party but it needs to be cancelled on behalf of someone